app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,89 @@
+class RentalsController < ApplicationController
+  before_action :find_video_and_customer, only: [:check_out, :check_in]
+
+  def check_out 
+    
+    if @video.nil? || @customer.nil? 
+      render json: {
+        errors: ['Not Found'],
+      }, status: :not_found 
+
+      return 
+    end 
```

Comment:
Since you repeat this logic I'd probably move it into `find_video_and_customer` as well.